src/Tribe/Field.php

```diff
@@ -340,6 +341,20 @@ public function doFieldName( $multi = false ) {
 		}
 
 		/**
+		 * returns the field's placeholder
+		 *
+		 * @return string the field value
+		 */
+		public function doFieldPlaceholder() {
```

Comment:
I'll make a ticket for the method deprecation so it can be done in a different PR.